ui/shared/ConfirmationDialog.qml

```diff
@@ -37,14 +39,11 @@ ModalPopup {
         width: parent.width
         height: children[0].height
 
-        StyledButton {
+        StatusButton {
             anchors.right: parent.right
             anchors.rightMargin: Style.current.smallPadding
-            btnColor: Style.current.lightRed
-            btnBorderWidth: 1
-            btnBorderColor: Style.current.grey
-            textColor: Style.current.red
-            label: confirmationDialog.confirmButtonLabel
+            color: Style.current.red
```

Comment:
`danger` uses `red` but it gives more context to the color use
```suggestion
            color: Style.current.danger
```

Reply:
That makes more sense! Fixed!